tests/reftests/source.test

```diff
@@ -48,17 +48,16 @@ pandore2/root.ml
 pandore2/src
 pandore2/src/src.ml
 ### rm -rf pandore.3
-### opam source pandore --dev | '\\' -> '/'
-Successfully fetched pandore development repo to ./pandore/
+### opam source pandore --dev
+Successfully fetched pandore development repo to ${BASEDIR}/pandore
```

Comment:
Do we want to keep the `./` for dev packages ?

Reply:
I think it’s fine as-is.